### DISCUSSION TITLE
Error, appeared when I adjusted the storage location.

### DISCUSSION THREAD

**Discussion by mugehan** [sayaka-sh/spmp#413] [Q&A]
---STACK TRACE---
java.lang.NullPointerException
	at com.toasterofbread.spmp.ui.layout.apppage.settingspage.category.SystemCategoryKt.getSystemCategoryItems$lambda$7$lambda$6$processDialogSelection(Unknown Source:95)
	at com.toasterofbread.spmp.ui.layout.apppage.settingspage.category.SystemCategoryKt.getSystemCategoryItems$lambda$7$lambda$6$processDialogSelection$default(Unknown Source:12)
	at com.toasterofbread.spmp.ui.layout.apppage.settingspage.category.SystemCategoryKt$getSystemCategoryItems$5$1$1.invokeSuspend(Unknown Source:25)
	at com.toasterofbread.spmp.ui.layout.apppage.settingspage.category.SystemCategoryKt$getSystemCategoryItems$5$1$1.invoke(SourceFile:2)
	at com.toasterofbread.spmp.ui.layout.apppage.settingspage.category.SystemCategoryKt$getSystemCategoryItems$5$1$1.invoke(SourceFile:1)
	at dev.toastbits.composekit.settings.ui.item.FileSettingsItem$Item$2$3$1$1.invokeSuspend(Unknown Source:37)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:8)
	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:112)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(Unknown Source:95)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@57f3889, Dispatchers.Default]

---LOGCAT (last 111 lines)---
12-10 19:58:50.853 22588 22644 I ViewRootImpl@e21853c[MainActivity]: Received frameDrawingCallback syncResult=0 frameNum=1.
12-10 19:58:50.853 22588 22644 I ViewRootImpl@e21853c[MainActivity]: mWNT: t=0xb4000072a6b31f10 mBlastBufferQueue=0xb4000070a6add0f0 fn= 1 mRenderHdrSdrRatio=1.0 caller= android.view.ViewRootImpl$8.onFrameDraw:13946 android.view.ThreadedRenderer$1.onFrameDraw:792 <bottom of call stack> 
12-10 19:58:50.853 22588 22644 I ViewRootImpl@e21853c[MainActivity]: Setting up sync and frameCommitCallback
12-10 19:58:50.855 22588 22624 I BLASTBufferQueue: [ViewRootImpl@e21853c[MainActivity]#6](f:0,a:0,s:0) onFrameAvailable the first frame is available
12-10 19:58:50.856 22588 22624 I ViewRootImpl@e21853c[MainActivity]: Received frameCommittedCallback lastAttemptedDrawFrameNum=1 didProduceBuffer=true
12-10 19:58:50.856 22588 22624 I SurfaceSyncGroup: onTransactionReady mName=wmsSync-ViewRootImpl@e21853c[MainActivity]#12 callback=39654559
12-10 19:58:50.856 22588 22588 I ViewRootImpl@e21853c[MainActivity]: reportDrawFinished seqId=0
12-10 19:58:50.862 22588 23016 I ViewRootImpl@8f1f1c0[MainActivity]: Resizing android.view.ViewRootImpl@5478b40: frame = [13,2539][1427,3120] reportDraw = true forceLayout = false syncSeqId = -1
12-10 19:58:50.863 22588 22588 I InsetsSourceConsumer: applyRequestedVisibilityToControl: visible=true, type=statusBars, host=com.toasterofbread.spmp/com.toasterofbread.spmp.MainActivity
12-10 19:58:50.863 22588 22588 I InsetsSourceConsumer: applyRequestedVisibilityToControl: visible=true, type=navigationBars, host=com.toasterofbread.spmp/com.toasterofbread.spmp.MainActivity
12-10 19:58:50.864 22588 22588 I BufferQueueProducer: [](id:583c00000007,api:0,p:32768,c:22588) setDequeueTimeout:2077252342
12-10 19:58:50.864 22588 22588 I BLASTBufferQueue_Java: new BLASTBufferQueue, mName= ViewRootImpl@8f1f1c0[MainActivity] mNativeObject= 0xb4000070a6ae7370 sc.mNativeObject= 0xb400007166af9a10 caller= android.view.ViewRootImpl.updateBlastSurfaceIfNeeded:3085 android.view.ViewRootImpl.relayoutWindow:10224 android.view.ViewRootImpl.performTraversals:4167 android.view.ViewRootImpl.doTraversal:3345 android.view.ViewRootImpl$TraversalRunnable.run:11437 android.view.Choreographer$CallbackRecord.run:1690 android.view.Choreographer$CallbackRecord.run:1699 android.view.Choreographer.doCallbacks:1154 android.view.Choreographer.doFrame:1080 android.view.Choreographer$FrameDisplayEventReceiver.run:1647 
12-10 19:58:50.864 22588 22588 I BLASTBufferQueue_Java: update, w= 1428 h= 595 mName = ViewRootImpl@8f1f1c0[MainActivity] mNativeObject= 0xb4000070a6ae7370 sc.mNativeObject= 0xb400007166af9a10 format= -2 caller= android.graphics.BLASTBufferQueue.<init>:89 android.view.ViewRootImpl.updateBlastSurfaceIfNeeded:3085 android.view.ViewRootImpl.relayoutWindow:10224 android.view.ViewRootImpl.performTraversals:4167 android.view.ViewRootImpl.doTraversal:3345 android.view.ViewRootImpl$TraversalRunnable.run:11437 
12-10 19:58:50.864 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: Relayout returned: old=(0,102,1440,3120) new=(13,2539,1427,3120) relayoutAsync=false req=(1414,581)0 dur=5 res=0x403 s={true 0xb400007276ae16b0} ch=true seqId=0
12-10 19:58:50.864 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: performConfigurationChange setNightDimText nightDimLevel=0
12-10 19:58:50.864 22588 22588 D ViewRootImpl@8f1f1c0[MainActivity]: mThreadedRenderer.initialize() mSurface={isValid=true 0xb400007276ae16b0} hwInitialized=true
12-10 19:58:50.866 22588 22588 D ViewRootImpl@8f1f1c0[MainActivity]: reportNextDraw android.view.ViewRootImpl.performTraversals:4781 android.view.ViewRootImpl.doTraversal:3345 android.view.ViewRootImpl$TraversalRunnable.run:11437 android.view.Choreographer$CallbackRecord.run:1690 android.view.Choreographer$CallbackRecord.run:1699 
12-10 19:58:50.866 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: Setup new sync=wmsSync-ViewRootImpl@8f1f1c0[MainActivity]#14
12-10 19:58:50.866 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: Creating new active sync group ViewRootImpl@8f1f1c0[MainActivity]#15
12-10 19:58:50.866 22588 22588 I SurfaceSyncGroup: addLocalSync=ViewRootImpl@8f1f1c0[MainActivity]#15 to name=wmsSync-ViewRootImpl@8f1f1c0[MainActivity]#14, callers=android.window.SurfaceSyncGroup.add:431 android.window.SurfaceSyncGroup.add:392 android.window.SurfaceSyncGroup.add:340 android.view.ViewRootImpl.createSyncIfNeeded:4912 android.view.ViewRootImpl.performTraversals:4796 android.view.ViewRootImpl.doTraversal:3345 
12-10 19:58:50.867 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: registerCallbacksForSync syncBuffer=false
12-10 19:58:50.867 22588 22643 I ViewRootImpl@8f1f1c0[MainActivity]: Received frameDrawingCallback syncResult=0 frameNum=1.
12-10 19:58:50.867 22588 22643 I ViewRootImpl@8f1f1c0[MainActivity]: mWNT: t=0xb4000072a6b26af0 mBlastBufferQueue=0xb4000070a6ae7370 fn= 1 mRenderHdrSdrRatio=1.0 caller= android.view.ViewRootImpl$8.onFrameDraw:13946 android.view.ThreadedRenderer$1.onFrameDraw:792 <bottom of call stack> 
12-10 19:58:50.867 22588 22643 I ViewRootImpl@8f1f1c0[MainActivity]: Setting up sync and frameCommitCallback
12-10 19:58:50.868 22588 22624 I BLASTBufferQueue: [ViewRootImpl@8f1f1c0[MainActivity]#7](f:0,a:0,s:0) onFrameAvailable the first frame is available
12-10 19:58:50.868 22588 22624 I ViewRootImpl@8f1f1c0[MainActivity]: Received frameCommittedCallback lastAttemptedDrawFrameNum=1 didProduceBuffer=true
12-10 19:58:50.869 22588 22624 I SurfaceSyncGroup: onTransactionReady mName=wmsSync-ViewRootImpl@8f1f1c0[MainActivity]#14 callback=229296911
12-10 19:58:50.869 22588 22624 D OpenGLRenderer: CFMS:: SetUp Pid : 22588    Tid : 22624
12-10 19:58:50.869 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: reportDrawFinished seqId=0
12-10 19:58:50.871 22588 22624 D OpenGLRenderer: HWUI - treat SMPTE_170M as sRGB
12-10 19:58:50.886 22588 23016 I ViewRootImpl@8f1f1c0[MainActivity]: Resizing android.view.ViewRootImpl@5478b40: frame = [13,2543][1427,3120] reportDraw = false forceLayout = false syncSeqId = -1
12-10 19:58:50.886 22588 22588 I BLASTBufferQueue_Java: update, w= 1428 h= 591 mName = ViewRootImpl@8f1f1c0[MainActivity] mNativeObject= 0xb4000070a6ae7370 sc.mNativeObject= 0xb400007166abaf50 format= -2 caller= android.view.ViewRootImpl.updateBlastSurfaceIfNeeded:3074 android.view.ViewRootImpl.relayoutWindow:10224 android.view.ViewRootImpl.performTraversals:4167 android.view.ViewRootImpl.doTraversal:3345 android.view.ViewRootImpl$TraversalRunnable.run:11437 android.view.Choreographer$CallbackRecord.run:1690 
12-10 19:58:50.886 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: Relayout returned: old=(13,2539,1427,3120) new=(13,2543,1427,3120) relayoutAsync=false req=(1414,577)0 dur=13 res=0x400 s={true 0xb400007276ae16b0} ch=false seqId=0
12-10 19:58:50.886 22588 22588 D ViewRootImpl@8f1f1c0[MainActivity]: mThreadedRenderer.updateSurface() mSurface={isValid=true 0xb400007276ae16b0}
12-10 19:58:50.887 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: registerCallbackForPendingTransactions
12-10 19:58:50.888 22588 22644 I ViewRootImpl@8f1f1c0[MainActivity]: mWNT: t=0xb4000072a6b203d0 mBlastBufferQueue=0xb4000070a6ae7370 fn= 2 mRenderHdrSdrRatio=1.0 caller= android.view.ViewRootImpl$6.onFrameDraw:5705 android.view.ViewRootImpl$2.onFrameDraw:2190 android.view.ThreadedRenderer$1.onFrameDraw:792 
12-10 19:58:50.888 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: handleResized, msg = 5 frames=ClientWindowFrames{frame=[13,2539][1427,3120] display=[0,102][1440,3120] parentFrame=[0,0][0,0]} forceNextWindowRelayout=false displayId=0 dragResizing=false compatScale=1.0 frameChanged=true attachedFrameChanged=false configChanged=false displayChanged=false compatScaleChanged=false
12-10 19:58:50.888 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: handleResized mSyncSeqId = 0
12-10 19:58:50.888 22588 22588 D ViewRootImpl@8f1f1c0[MainActivity]: reportNextDraw android.view.ViewRootImpl.handleResized:2574 android.view.ViewRootImpl.-$$Nest$mhandleResized:0 android.view.ViewRootImpl$ViewRootHandler.handleMessageImpl:7267 android.view.ViewRootImpl$ViewRootHandler.handleMessage:7236 android.os.Handler.dispatchMessage:106 
12-10 19:58:50.890 22588 22588 I BLASTBufferQueue_Java: update, w= 1428 h= 591 mName = ViewRootImpl@8f1f1c0[MainActivity] mNativeObject= 0xb4000070a6ae7370 sc.mNativeObject= 0xb400007166abaf50 format= -2 caller= android.view.ViewRootImpl.updateBlastSurfaceIfNeeded:3074 android.view.ViewRootImpl.relayoutWindow:10224 android.view.ViewRootImpl.performTraversals:4167 android.view.ViewRootImpl.doTraversal:3345 android.view.ViewRootImpl$TraversalRunnable.run:11437 android.view.Choreographer$CallbackRecord.run:1690 
12-10 19:58:50.890 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: Relayout returned: old=(13,2539,1427,3120) new=(13,2543,1427,3120) relayoutAsync=true req=(1414,577)0 dur=0 res=0x0 s={true 0xb400007276ae16b0} ch=false seqId=0
12-10 19:58:50.890 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: Setup new sync=wmsSync-ViewRootImpl@8f1f1c0[MainActivity]#16
12-10 19:58:50.890 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: Creating new active sync group ViewRootImpl@8f1f1c0[MainActivity]#17
12-10 19:58:50.890 22588 22588 I SurfaceSyncGroup: addLocalSync=ViewRootImpl@8f1f1c0[MainActivity]#17 to name=wmsSync-ViewRootImpl@8f1f1c0[MainActivity]#16, callers=android.window.SurfaceSyncGroup.add:431 android.window.SurfaceSyncGroup.add:392 android.window.SurfaceSyncGroup.add:340 android.view.ViewRootImpl.createSyncIfNeeded:4912 android.view.ViewRootImpl.performTraversals:4796 android.view.ViewRootImpl.doTraversal:3345 
12-10 19:58:50.891 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: registerCallbacksForSync syncBuffer=false
12-10 19:58:50.891 22588 22643 I ViewRootImpl@8f1f1c0[MainActivity]: Received frameDrawingCallback syncResult=0 frameNum=3.
12-10 19:58:50.891 22588 22643 I ViewRootImpl@8f1f1c0[MainActivity]: mWNT: t=0xb4000072a6ae3230 mBlastBufferQueue=0xb4000070a6ae7370 fn= 3 mRenderHdrSdrRatio=1.0 caller= android.view.ViewRootImpl$8.onFrameDraw:13946 android.view.ThreadedRenderer$1.onFrameDraw:792 <bottom of call stack> 
12-10 19:58:50.891 22588 22643 I ViewRootImpl@8f1f1c0[MainActivity]: Setting up sync and frameCommitCallback
12-10 19:58:50.892 22588 22624 I ViewRootImpl@8f1f1c0[MainActivity]: Received frameCommittedCallback lastAttemptedDrawFrameNum=3 didProduceBuffer=true
12-10 19:58:50.892 22588 22624 I SurfaceSyncGroup: onTransactionReady mName=wmsSync-ViewRootImpl@8f1f1c0[MainActivity]#16 callback=13303023
12-10 19:58:50.892 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: reportDrawFinished seqId=0
12-10 19:58:50.893 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: handleResized, msg = 4 frames=ClientWindowFrames{frame=[13,2543][1427,3120] display=[0,102][1440,3120] parentFrame=[0,0][0,0]} forceNextWindowRelayout=false displayId=0 dragResizing=false compatScale=1.0 frameChanged=false attachedFrameChanged=false configChanged=false displayChanged=false compatScaleChanged=false
12-10 19:58:50.904 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: handleWindowFocusChanged: 1 0 call from android.view.ViewRootImpl.-$$Nest$mhandleWindowFocusChanged:0
12-10 19:58:50.905 22588 22588 D ViewRootImpl@8f1f1c0[MainActivity]: mThreadedRenderer.initializeIfNeeded()#2 mSurface={isValid=true 0xb400007276ae16b0}
12-10 19:58:50.905 22588 22588 D InputMethodManagerUtils: startInputInner - Id : 0
12-10 19:58:50.905 22588 22588 I InputMethodManager: startInputInner - IInputMethodManagerGlobalInvoker.startInputOrWindowGainedFocus
12-10 19:58:50.919 22588 23143 I ViewRootImpl@e21853c[MainActivity]: Resizing android.view.ViewRootImpl@c438933: frame = [0,0][1440,3120] reportDraw = false forceLayout = false syncSeqId = -1
12-10 19:58:50.919 22588 22608 I ViewRootImpl@8f1f1c0[MainActivity]: Resizing android.view.ViewRootImpl@5478b40: frame = [13,2543][1427,3120] reportDraw = false forceLayout = false syncSeqId = -1
12-10 19:58:50.922 22588 22588 I InsetsSourceConsumer: applyRequestedVisibilityToControl: visible=false, type=ime, host=com.toasterofbread.spmp/com.toasterofbread.spmp.MainActivity
12-10 19:58:50.923 22588 22588 I ViewRootImpl@e21853c[MainActivity]: handleResized, msg = 4 frames=ClientWindowFrames{frame=[0,0][1440,3120] display=[-100000,-100000][100000,100000] parentFrame=[0,0][0,0]} forceNextWindowRelayout=false displayId=0 dragResizing=false compatScale=1.0 frameChanged=false attachedFrameChanged=false configChanged=false displayChanged=false compatScaleChanged=false
12-10 19:58:50.923 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: handleResized, msg = 4 frames=ClientWindowFrames{frame=[13,2543][1427,3120] display=[0,102][1440,3120] parentFrame=[0,0][0,0]} forceNextWindowRelayout=false displayId=0 dragResizing=false compatScale=1.0 frameChanged=false attachedFrameChanged=false configChanged=false displayChanged=false compatScaleChanged=false
12-10 19:58:51.693 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: ViewPostIme pointer 0
12-10 19:58:51.757 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: ViewPostIme pointer 1
12-10 19:58:51.837 22588 32172 W System.err: java.lang.NullPointerException
12-10 19:58:51.837 22588 32172 W System.err: 	at com.toasterofbread.spmp.ui.layout.apppage.settingspage.category.SystemCategoryKt.getSystemCategoryItems$lambda$7$lambda$6$processDialogSelection(Unknown Source:95)
12-10 19:58:51.837 22588 32172 W System.err: 	at com.toasterofbread.spmp.ui.layout.apppage.settingspage.category.SystemCategoryKt.getSystemCategoryItems$lambda$7$lambda$6$processDialogSelection$default(Unknown Source:12)
12-10 19:58:51.837 22588 32172 W System.err: 	at com.toasterofbread.spmp.ui.layout.apppage.settingspage.category.SystemCategoryKt$getSystemCategoryItems$5$1$1.invokeSuspend(Unknown Source:25)
12-10 19:58:51.837 22588 32172 W System.err: 	at com.toasterofbread.spmp.ui.layout.apppage.settingspage.category.SystemCategoryKt$getSystemCategoryItems$5$1$1.invoke(SourceFile:2)
12-10 19:58:51.837 22588 32172 W System.err: 	at com.toasterofbread.spmp.ui.layout.apppage.settingspage.category.SystemCategoryKt$getSystemCategoryItems$5$1$1.invoke(SourceFile:1)
12-10 19:58:51.837 22588 32172 W System.err: 	at dev.toastbits.composekit.settings.ui.item.FileSettingsItem$Item$2$3$1$1.invokeSuspend(Unknown Source:37)
12-10 19:58:51.837 22588 32172 W System.err: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(Unknown Source:8)
12-10 19:58:51.837 22588 32172 W System.err: 	at kotlinx.coroutines.DispatchedTask.run(Unknown Source:112)
12-10 19:58:51.837 22588 32172 W System.err: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(Unknown Source:95)
12-10 19:58:51.837 22588 32172 W System.err: 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@57f3889, Dispatchers.Default]
12-10 19:58:51.860 22588 22588 W SQLiteLog: (28) double-quoted string literal: "version"
12-10 19:58:51.860 22588 22588 W SQLiteLog: (28) double-quoted string literal: "Version"
12-10 19:58:51.861 22588 22588 I System.out: Database is already up to date (version 6)
12-10 19:58:51.861 22588 32172 I System.out: Retrieving logcat output...
12-10 19:58:51.862 22588 22588 I DecorView: setWindowBackground: isPopOver=false color=ff17171a d=android.graphics.drawable.GradientDrawable@c6bcb31
12-10 19:58:51.864 22588 22624 D NativeCustomFrequencyManager: [NativeCFMS] BpCustomFrequencyManager::BpCustomFrequencyManager()
12-10 19:58:51.868 22588 22588 I InsetsController: onStateChanged: host=com.toasterofbread.spmp/com.toasterofbread.spmp.ErrorReportActivity, from=android.view.ViewRootImpl.setView:1797, state=InsetsState: {mDisplayFrame=Rect(0, 0 - 1440, 3120), mDisplayCutout=DisplayCutout{insets=Rect(0, 101 - 0, 0) waterfall=Insets{left=0, top=0, right=0, bottom=0} boundingRect={Bounds=[Rect(0, 0 - 0, 0), Rect(687, 0 - 754, 101), Rect(0, 0 - 0, 0), Rect(0, 0 - 0, 0)]} cutoutPathParserInfo={CutoutPathParserInfo{displayWidth=1440 displayHeight=3120 physicalDisplayWidth=1440 physicalDisplayHeight=3120 density={3.75} cutoutSpec={M 0,0 H -8.933333333333333 V 26.93333333333333 H 8.933333333333333 V 0 H 0 Z @dp} rotation={0} scale={1.0} physicalPixelDisplaySizeRatio={1.0}}}}, mRoundedCorners=RoundedCorners{[RoundedCorner{position=TopLeft, radius=7, center=Point(7, 7)}, RoundedCorner{position=TopRight, radius=7, center=Point(1433, 7)}, RoundedCorner{position=BottomRight, radius=7, center=Point(1433, 3113)}, RoundedCorner{position=BottomLeft, radius=7, center=Point(7, 3113)}]}  mRoundedCornerFrame=Rect(0, 0 - 1440, 3120), mPrivacyIndicatorBounds=PrivacyIndicatorBounds {static bounds=Rect(1293, 0 - 1440, 102) rotation=0}, mDisplayShape=DisplayShape{ spec=-1951251426 displayWidth=1440 displayHeight=3120 physicalPixelDisplaySizeRatio=1.0 rotation=0 offsetX=0 offsetY=0 scale=1.0}, mSources= { InsetsSource: {b2d00001 mType=navigationBars mFrame=[0,0][0,0] mVisible=true mFlags=[SUPPRESS_SCRIM]}, InsetsSource: {b2d00004 mType=systemGestures mFrame=[0,0][75,3120] mVisible=true mFlags=[]}, InsetsSource: {b2d00005 mType=mandatorySystemGestures mFrame=[0,3013][1440,3120] mVisible=true mFlags=[]}, InsetsSource: {b2d00006 mType=tappableElement mFrame=[0,0][0,0] mVisible=true mFlags=[]}, InsetsSource: {b2d00024 mType=systemGestures mFrame=[1365,0][1440,3120] mVisible=true mFlags=[]}, InsetsSource: {df340000 mType=statusBars mFrame=[0,0][1440,102] mVisible=true mFlags=[]}, InsetsSource: {df340005 mType=mandatorySystemGestures mFrame=[0,0][1440,141] mVisible=true mFlags=[]}, InsetsSource: {df340006 mType=tappableElement mFrame=[0,0][1440,102] mVisible=true mFlags=[]}, InsetsSource: {27 mType=displayCutout mFrame=[0,0][1440,101] mVisible=true mFlags=[]} }
12-10 19:58:51.868 22588 22588 I ViewRootImpl@371491c[ErrorReportActivity]: synced displayState. AttachInfo displayState=2
12-10 19:58:51.870 22588 22588 I ViewRootImpl@371491c[ErrorReportActivity]: setView = com.android.internal.policy.DecorView@a7968ab TM=true
12-10 19:58:51.877 22588 22588 I InsetsController: onStateChanged: host=com.toasterofbread.spmp/com.toasterofbread.spmp.ErrorReportActivity, from=android.view.ViewRootImpl.relayoutWindow:10165, state=InsetsState: {mDisplayFrame=Rect(0, 0 - 1440, 3120), mDisplayCutout=DisplayCutout{insets=Rect(0, 101 - 0, 0) waterfall=Insets{left=0, top=0, right=0, bottom=0} boundingRect={Bounds=[Rect(0, 0 - 0, 0), Rect(687, 0 - 754, 101), Rect(0, 0 - 0, 0), Rect(0, 0 - 0, 0)]} cutoutPathParserInfo={CutoutPathParserInfo{displayWidth=1440 displayHeight=3120 physicalDisplayWidth=1440 physicalDisplayHeight=3120 density={3.75} cutoutSpec={M 0,0 H -8.933333333333333 V 26.93333333333333 H 8.933333333333333 V 0 H 0 Z @dp} rotation={0} scale={1.0} physicalPixelDisplaySizeRatio={1.0}}}}, mRoundedCorners=RoundedCorners{[RoundedCorner{position=TopLeft, radius=7, center=Point(7, 7)}, RoundedCorner{position=TopRight, radius=7, center=Point(1433, 7)}, RoundedCorner{position=BottomRight, radius=7, center=Point(1433, 3113)}, RoundedCorner{position=BottomLeft, radius=7, center=Point(7, 3113)}]}  mRoundedCornerFrame=Rect(0, 0 - 1440, 3120), mPrivacyIndicatorBounds=PrivacyIndicatorBounds {static bounds=Rect(1293, 0 - 1440, 102) rotation=0}, mDisplayShape=DisplayShape{ spec=-1951251426 displayWidth=1440 displayHeight=3120 physicalPixelDisplaySizeRatio=1.0 rotation=0 offsetX=0 offsetY=0 scale=1.0}, mSources= { InsetsSource: {b2d00001 mType=navigationBars mFrame=[0,0][0,0] mVisible=true mFlags=[SUPPRESS_SCRIM]}, InsetsSource: {b2d00004 mType=systemGestures mFrame=[0,0][75,3120] mVisible=true mFlags=[]}, InsetsSource: {b2d00005 mType=mandatorySystemGestures mFrame=[0,3013][1440,3120] mVisible=true mFlags=[]}, InsetsSource: {b2d00006 mType=tappableElement mFrame=[0,0][0,0] mVisible=true mFlags=[]}, InsetsSource: {b2d00024 mType=systemGestures mFrame=[1365,0][1440,3120] mVisible=true mFlags=[]}, InsetsSource: {df340000 mType=statusBars mFrame=[0,0][1440,102] mVisible=true mFlags=[]}, InsetsSource: {df340005 mType=mandatorySystemGestures mFrame=[0,0][1440,141] mVisible=true mFlags=[]}, InsetsSource: {df340006 mType=tappableElement mFrame=[0,0][1440,102] mVisible=true mFlags=[]}, InsetsSource: {3 mType=ime mFrame=[0,0][0,0] mVisible=false mFlags=[]}, InsetsSource: {27 mType=displayCutout mFrame=[0,0][1440,101] mVisible=true mFlags=[]} }
12-10 19:58:51.877 22588 22588 I BufferQueueProducer: [](id:583c00000008,api:0,p:0,c:22588) setDequeueTimeout:2077252342
12-10 19:58:51.877 22588 22588 I BLASTBufferQueue_Java: new BLASTBufferQueue, mName= ViewRootImpl@371491c[ErrorReportActivity] mNativeObject= 0xb4000070a6ae34f0 sc.mNativeObject= 0xb400007166aadc90 caller= android.view.ViewRootImpl.updateBlastSurfaceIfNeeded:3085 android.view.ViewRootImpl.relayoutWindow:10224 android.view.ViewRootImpl.performTraversals:4167 android.view.ViewRootImpl.doTraversal:3345 android.view.ViewRootImpl$TraversalRunnable.run:11437 android.view.Choreographer$CallbackRecord.run:1690 android.view.Choreographer$CallbackRecord.run:1699 android.view.Choreographer.doCallbacks:1154 android.view.Choreographer.doFrame:1080 android.view.Choreographer$FrameDisplayEventReceiver.run:1647 
12-10 19:58:51.877 22588 22588 I BLASTBufferQueue_Java: update, w= 1440 h= 3120 mName = ViewRootImpl@371491c[ErrorReportActivity] mNativeObject= 0xb4000070a6ae34f0 sc.mNativeObject= 0xb400007166aadc90 format= -1 caller= android.graphics.BLASTBufferQueue.<init>:89 android.view.ViewRootImpl.updateBlastSurfaceIfNeeded:3085 android.view.ViewRootImpl.relayoutWindow:10224 android.view.ViewRootImpl.performTraversals:4167 android.view.ViewRootImpl.doTraversal:3345 android.view.ViewRootImpl$TraversalRunnable.run:11437 
12-10 19:58:51.878 22588 22588 I ViewRootImpl@371491c[ErrorReportActivity]: Relayout returned: old=(0,0,1440,3120) new=(0,0,1440,3120) relayoutAsync=false req=(1440,3120)0 dur=4 res=0x403 s={true 0xb400007276af6b50} ch=true seqId=0
12-10 19:58:51.878 22588 22588 I ViewRootImpl@371491c[ErrorReportActivity]: performConfigurationChange setNightDimText nightDimLevel=0
12-10 19:58:51.878 22588 22588 D ViewRootImpl@371491c[ErrorReportActivity]: mThreadedRenderer.initialize() mSurface={isValid=true 0xb400007276af6b50} hwInitialized=true
12-10 19:58:51.878 22588 22588 D ViewRootImpl@371491c[ErrorReportActivity]: reportNextDraw android.view.ViewRootImpl.performTraversals:4781 android.view.ViewRootImpl.doTraversal:3345 android.view.ViewRootImpl$TraversalRunnable.run:11437 android.view.Choreographer$CallbackRecord.run:1690 android.view.Choreographer$CallbackRecord.run:1699 
12-10 19:58:51.878 22588 22588 I ViewRootImpl@371491c[ErrorReportActivity]: Setup new sync=wmsSync-ViewRootImpl@371491c[ErrorReportActivity]#18
12-10 19:58:51.878 22588 22588 I ViewRootImpl@371491c[ErrorReportActivity]: Creating new active sync group ViewRootImpl@371491c[ErrorReportActivity]#19
12-10 19:58:51.879 22588 22588 I SurfaceSyncGroup: addLocalSync=ViewRootImpl@371491c[ErrorReportActivity]#19 to name=wmsSync-ViewRootImpl@371491c[ErrorReportActivity]#18, callers=android.window.SurfaceSyncGroup.add:431 android.window.SurfaceSyncGroup.add:392 android.window.SurfaceSyncGroup.add:340 android.view.ViewRootImpl.createSyncIfNeeded:4912 android.view.ViewRootImpl.performTraversals:4796 android.view.ViewRootImpl.doTraversal:3345 
12-10 19:58:51.879 22588 22588 I ViewRootImpl@371491c[ErrorReportActivity]: registerCallbacksForSync syncBuffer=false
12-10 19:58:51.880 22588 22643 I ViewRootImpl@371491c[ErrorReportActivity]: Received frameDrawingCallback syncResult=0 frameNum=1.
12-10 19:58:51.880 22588 22643 I ViewRootImpl@371491c[ErrorReportActivity]: mWNT: t=0xb4000072a6b2e3f0 mBlastBufferQueue=0xb4000070a6ae34f0 fn= 1 mRenderHdrSdrRatio=1.0 caller= android.view.ViewRootImpl$8.onFrameDraw:13946 android.view.ThreadedRenderer$1.onFrameDraw:792 <bottom of call stack> 
12-10 19:58:51.880 22588 22643 I ViewRootImpl@371491c[ErrorReportActivity]: Setting up sync and frameCommitCallback
12-10 19:58:51.880 22588 22624 I BLASTBufferQueue: [ViewRootImpl@371491c[ErrorReportActivity]#8](f:0,a:0,s:0) onFrameAvailable the first frame is available
12-10 19:58:51.880 22588 22624 I ViewRootImpl@371491c[ErrorReportActivity]: Received frameCommittedCallback lastAttemptedDrawFrameNum=1 didProduceBuffer=true
12-10 19:58:51.880 22588 22624 I SurfaceSyncGroup: onTransactionReady mName=wmsSync-ViewRootImpl@371491c[ErrorReportActivity]#18 callback=240225795
12-10 19:58:51.881 22588 22624 D OpenGLRenderer: CFMS:: SetUp Pid : 22588    Tid : 22624
12-10 19:58:51.881 22588 22588 I ViewRootImpl@371491c[ErrorReportActivity]: reportDrawFinished seqId=0
12-10 19:58:51.881 22588 22588 I InsetsSourceConsumer: applyRequestedVisibilityToControl: visible=true, type=statusBars, host=com.toasterofbread.spmp/com.toasterofbread.spmp.MainActivity
12-10 19:58:51.882 22588 22588 I InsetsSourceConsumer: applyRequestedVisibilityToControl: visible=true, type=navigationBars, host=com.toasterofbread.spmp/com.toasterofbread.spmp.MainActivity
12-10 19:58:51.882 22588 22624 D OpenGLRenderer: HWUI - treat SMPTE_170M as sRGB
12-10 19:58:51.882 22588 22588 I ViewRootImpl@8f1f1c0[MainActivity]: handleWindowFocusChanged: 0 0 call from android.view.ViewRootImpl.-$$Nest$mhandleWindowFocusChanged:0
12-10 19:58:51.882 22588 22588 I ImeFocusController: onPreWindowFocus: skipped, hasWindowFocus=false mHasImeFocus=true
12-10 19:58:51.882 22588 22588 I ImeFocusController: onPostWindowFocus: skipped, hasWindowFocus=false mHasImeFocus=true
12-10 19:58:51.887 22588 22588 I InsetsSourceConsumer: applyRequestedVisibilityToControl: visible=true, type=statusBars, host=com.toasterofbread.spmp/com.toasterofbread.spmp.ErrorReportActivity
12-10 19:58:51.888 22588 22588 I InsetsSourceConsumer: applyRequestedVisibilityToControl: visible=true, type=navigationBars, host=com.toasterofbread.spmp/com.toasterofbread.spmp.ErrorReportActivity

**Comment by sayaka-sh** [comment]
Please [create a bug report](https://github.com/toasterofbread/spmp/issues/new?assignees=&labels=Bug&projects=&template=bug_report.yml).
